src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -99,14 +122,13 @@ g.test('importExternalTexture,sample')
   .desc(
     `
 Tests that we can import an HTMLVideoElement into a GPUExternalTexture, sample from it for all
-supported video formats {vp8, vp9, ogg, mp4}, and ensure the GPUExternalTexture is destroyed by
-a microtask.
-TODO: Multiplanar scenarios
+supported video formats {vp8, vp9, ogg}, common source colorspaces {bt.601, bt.709, bt.2020},
+and to ensure the GPUExternalTexture is destroyed by a microtask.
```

Comment:
Note that we don't need to ensure GPUExternalTexture is destroyed by a microtask. It is not true for current spec any more.

Reply:
Good catch, I didn't think about this test description very closely.

I assume this test still tests that it's destroyed in a microtask, so please add a TODO here and then we should actually fix the test separately.

Reply:
This test doesn't actually test the destroy behavior - my guess is that I decided to split the test out in a prior commit. It looks like Shaobo is already revising it, so I'll just remove it from the description and leave the destroy test as is.